Resources/Prototypes/Entities/Mobs/NPCs/bear.yml

```diff
@@ -70,11 +85,14 @@
         path: /Audio/Weapons/Xeno/alien_claw_flesh3.ogg
     angle: 0
     animation: WeaponArcClaw
+    bluntStaminaDamageFactor: 3.0
     damage:
-      groups:
-        Brute: 15
+      types:
+        Blunt: 10
+        Slash: 5
   - type: ReplacementAccent
     accent: genericAggressive
+  #remove
```

Comment:
comment...?

Reply:
Oh that's right....I was thinking about removing the ghost takeover since the Salvage variant already has it? Making this version NPC only.

Reply:
yeah that's good